govcd/api_vcd_test.go

```diff
@@ -142,6 +142,8 @@ type TestConfig struct {
 	OVA struct {
 		OVAPath        string `yaml:"ovaPath,omitempty"`
 		OVAChunkedPath string `yaml:"ovaChunkedPath,omitempty"`
+		OvaMultiVmPath string `yaml:"ovaMultiVmPath,omitempty"`
+		OvaVmName      string `yaml:"ovaVmName,omitempty"`
```

Comment:
"OVA" vs "Ova" inconsistency. Let's rename the `OVAPath` and `OVAChunkedPath` to use the new "Ova..." prefix.

Reply:
done